src/libs/fileDownload/index.js

```diff
@@ -5,38 +5,43 @@ import getAttachmentName from './getAttachmentName';
  * Downloading attachment in web, desktop
  * @param {String} url
  * @param {String} fileName
+ * @returns {Promise}
  */
 export default function fileDownload(url, fileName) {
-    fetch(url)
-        .then(response => response.blob())
-        .then((blob) => {
+    return new Promise((resolve) => {
+        fetch(url)
+            .then(response => response.blob())
+            .then((blob) => {
             // Create blob link to download
```

Comment:
Indent this as well

Reply:
Add an indent in the second `then` block? I tried adding an indent here and `eslint` automatically removes the indent.